civis_jupyter_notebooks/assets/civis_client_config.py

```diff
@@ -4,7 +4,7 @@
 
 if 'CIVIS_API_KEY' in os.environ:
     LOGGER.info('creating civis api client')
-    client = civis.APIClient(resources='all')
+    client = civis.APIClient()
```

Comment:
I also removed the deprecated `resources='all'` argument from our instantiations of the API client.  It was deprecated in civisanalytics/civis-python#243.

Reply:
Ah yeah that's a good point.  I'll revert this change, because there's no real reason to break compatibility with `civis` versions before 1.9.

Reply:
Hmm, [1.9](https://github.com/civisanalytics/civis-python/releases/tag/v1.9.0) itself is already over 2 years old. Maybe a question for python client maintainers, but do we really want to keep supporting versions that old? I imagine there are plans to fully remove the `resources` parameter, given the deprecation warnings, so we will probably need to make this change eventually

Reply:
Yeah, I agree, we should ask the maintainers.  I think it's out of scope for this PR though.

Reply:
Discussed this offline; we'll bump the required version to 1.9.